packages/babel-plugin-proposal-decorators/test/fixtures/ordering/decorators/exec.js

```diff
@@ -0,0 +1,31 @@
+var log = [];
+
+function push(x) { log.push(x); return x; }
+
+function logFinisher(a, b) {
```

Comment:
This has to do with the execution of the decorator, not the finisher; consider renaming this test and adding other tests to check the ordering of finishers.

Reply:
> adding other tests to check the ordering of finishers

https://github.com/babel/babel/pull/7976/files/ce8a1818c9992377ed26767bd0336b20e6bd5318#diff-3a4ca738a2b3ce6ab485658c51556f68